src/deploy/standalone/logrotate_noobaa.conf

```
@@ -1,46 +1,41 @@
 /var/log/noobaa.log
 {
-    daily
     size 100M
     start 1
     missingok
     rotate 100
     compress
     create 644 root root
     sharedscripts
-    postrotate
-        /bin/kill -HUP `cat /var/run/syslogd.pid 2> /dev/null` 2> /dev/null || true
-        /bin/kill -HUP `cat /var/run/rsyslogd.pid 2> /dev/null` 2> /dev/null || true
+    postrotate 
+        killall -HUP rsyslogd || true
     endscript
 }
 
-/var/log/client_noobaa.log
+/var/log/noobaa_events.log
 {
-    daily
-    size 100M
+    size 10M
     start 1
     missingok
-    rotate 10
+    rotate 100
     compress
     create 644 root root
     sharedscripts
     postrotate
-        /bin/kill -HUP `cat /var/run/syslogd.pid 2> /dev/null` 2> /dev/null || true
-        /bin/kill -HUP `cat /var/run/rsyslogd.pid 2> /dev/null` 2> /dev/null || true
+        killall -HUP rsyslogd || true
     endscript
 }
 
-/var/log/noobaa_events.log
+/var/log/client_noobaa.log
 {
-    monthly
-    size 10M
+    size 100M
     start 1
     missingok
     rotate 10
+    compress
     create 644 root root
     sharedscripts
     postrotate
-        /bin/kill -HUP `cat /var/run/syslogd.pid 2> /dev/null` 2> /dev/null || true
-        /bin/kill -HUP `cat /var/run/rsyslogd.pid 2> /dev/null` 2> /dev/null || true
+        killall -HUP rsyslogd || true
```

btw now that I look at this - is there a reason we removed sending the HUP signal to syslogd? did that cause any trouble or can we keep both? I think there are systems that still run syslog daemon as syslogd (eg [AIX](https://www.ibm.com/docs/en/aix/7.3?topic=s-syslogd-daemon))
```suggestion
        killall -HUP rsyslogd || true
        killall -HUP syslogd || true
```

---

from local testing what I can see is It didnt cause any issue new logs are getting added to newly created log files